src/python/grpcio/grpc/__init__.py

```diff
@@ -833,7 +833,11 @@ class HandlerCallDetails(six.with_metaclass(abc.ABCMeta)):
 
 
 class GenericRpcHandler(six.with_metaclass(abc.ABCMeta)):
```

Comment:
I still think that type inspection is favorable over introducing API complexity.  Also, we are introducing a potential quagmire of versioning issues with generated code vs library version.  If anyone uses a newer version of grpc-tools and an older version of gRPC, they will get a bunch of weird attribute exceptions.